content/docs/command-reference/params/index.md

```diff
@@ -16,62 +16,70 @@ positional arguments:
 
 ## Description
 
-In order to track parameters and hyperparameters associated to machine learning
-experiments in <abbr>DVC projects</abbr>, DVC provides a different type of
-dependencies: _parameters_. They usually have simple names like `epochs`,
-`learning-rate`, `batch_size`, etc.
+Parameters are simple values used inside your code to influence the results
```

Comment:
`simple` - I still think is not appropriate 🤔

Reply:
I removed "simple" everywhere.